src/js/health-records/containers/DownloadPage.jsx

```diff
@@ -12,33 +12,74 @@ import DownloadLink from '../components/DownloadLink';
+          };
+          break;
+        case 'DodMilitaryService':
+          alertProps = {
+            headline: 'Your Department of Defense information may not be current.',
+            content: (<p>Most of your health record should be up to date. But we’re having trouble accessing your Department of Defense (DoD) information right now. Please refresh this page or try again later. If you still don’t see your updated DoD information, please call the Vets.gov Help Desk at 855-574-7286 (TTY: 800-829-4833). We’re here Monday–Friday, 8:00 a.m.–8:00 p.m. (ET). If you need this information right away, please call your VA health care provider.</p>)
+          };
+          break;
+        default: // Keep default message.
+      }
+    }
+
+    return <AlertBox isVisible status="warning" {...alertProps}/>;
+  }
+
+  renderMessageBanner = () => {
+    const { form, refresh } = this.props;
+    let alertProps;
+
     if (!form.ready) {
-      alertProps.headline = (<h4>Couldn’t generate your records</h4>);
-      alertProps.content = (<p>Unfortunately, we weren’t able to generate your health records. Please try again later. You can also call the Vets.gov Help Desk at <a href="tel:855-574-7286">1-855-574-7286</a>, TTY: <a href="tel:18008778339">1-800-877-8339</a>, Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET).</p>);
-      alertProps.status = 'error';
+      alertProps = {
+        headline: 'Couldn’t generate your records',
+        content: (<p>Unfortunately, we weren’t able to generate your health records. Please try again later. You can also call the Vets.gov Help Desk at <a href="tel:855-574-7286">1-855-574-7286</a>, TTY: <a href="tel:18008778339">1-800-877-8339</a>, Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET).</p>),
+        status: 'error'
+      };
     } else if (refresh && !isEmpty(refresh.statuses.incomplete)) {
-      alertProps.headline = (<h4>Your health records are not up to date</h4>);
-      alertProps.content = (<p>This older version of your health records may have outdated or missing information.</p>
-      );
-      alertProps.status = 'warning';
+      return this.renderIncompleteMessage();
     } else if (refresh && !isEmpty(refresh.statuses.failed)) {
-      alertProps.headline = (<h4>Couldn’t update your records</h4>);
-      alertProps.content = (<p>Unfortunately, we weren’t able to generate your most recent health records. You can try again in 24 hours or download an older version of your records below.</p>);
-      alertProps.status = 'warning';
+      alertProps = {
+        headline: 'Couldn’t update your records',
```

Comment:
"Your records couldn't be updated?" for consistency maybe? This seems like an incomplete sentence.